src/styles/common.scss

```diff
@@ -265,5 +266,12 @@
     gap: 4px;
     width: 100%;
     margin-bottom: 5%;
-    aspect-ratio: 2.26;
+}
+
+@media (max-width: $tablet-width) {
+    .rowOfCards {
+        grid-auto-rows: minmax(0, 1fr);
+        grid-auto-flow: row;
```

Comment:
I stay with `grid`, because `display: block` doesn't give me `gap` between cards.

Reply:
I don't understand.

Reply:
I see what happened here. The actual Wix website built on this template doesn't have any wrapping at mobile width:

<img width="396" alt="image" src="https://github.com/user-attachments/assets/bdcde0a0-8e28-4aa6-bbf8-4d5a44200560">

Reply:
It only wraps when you reload the page. So I didn't understand what kind of stying you're trying to achieve.

Reply:
I think you should use flex in both cases and change flex-direction from row to column.

Reply:
Put aspect-ratio on the direct children of rowOfCards. Rename rowOfCards to something cardGroup.